clients/python/game/models.py

```diff
@@ -0,0 +1,18 @@
+class Game:
+    def __init__(self, id_, tick, players, resources, map_, viruses):
+        self.id = id_
+        self.tick = tick
+        self.players = players
+        self.resources = resources
+        self.map = map_
+        self.viruses = viruses
```

Comment:
Since we currently have nothing planned (no issue I think) for the viruses, do we want to put it here?